live-examples/css-examples/pseudo-class/meta.json

```
@@ -153,6 +180,33 @@
             "defaultTab": "css",
             "height": "tabbed-shorter"
         },
+        "optional": {
+            "cssExampleSrc": "./live-examples/css-examples/pseudo-class/optional.css",
+            "exampleCode": "./live-examples/css-examples/pseudo-class/optional.html",
+            "fileName": "pseudo-class-optional.html",
+            "title": "CSS Demo: :optional",
+            "type": "tabbed",
+            "defaultTab": "css",
+            "height": "tabbed-shorter"
```

```suggestion
            "height": "tabbed-standard"
```

(the HTML overflows)